src/main/api/studio-api.yaml

```diff
@@ -10327,6 +10298,92 @@ components:
         - publishingTarget
 
     PublishingPackageDetails:
+      allOf:
+        - $ref: '#/components/schemas/PublishingPackageStatus'
+        - type: object
+          properties:
+            items:
+              type: array
+              items:
+                type: object
+                properties:
+                  path:
+                    type: string
+                    description: content item path
+                  contentTypeClass:
+                    type: string
+                    description: content type class
+                  mimeType:
+                    type: string
+                    description: mime-type
+          required:
+            - id
+            - schedule
+            - approver
+            - state
+            - publishingTarget
+            - items
+
+    PublishingProgress:
+      type: object
+      properties:
+        startTime:
+            type: string
+            format: date-time
+            description: start time of the package processing
+        stages:
+          example:
+            - name: "Repo sync"
+              state: "COMPLETED"
+              processed: 1
+              total: 1
+              errors: false
+            - name: "Item list loading"
+              state: "COMPLETED"
+              processed: 100
+              total: 100
+              errors: false
+            - name: "Live publish"
```

Comment:
Let's add staging publish here (we can indicate that it's optional)

Reply:
Done